### ISSUE TITLE
Add support for Cloudflare workers-ai models

### ISSUE THREAD

**Issue by zzantares** [karthink/gptel#504]
Hi there! this is a request to add support for https://developers.cloudflare.com/workers-ai/get-started/rest-api/ it provides many models behind a single interface https://developers.cloudflare.com/workers-ai/models/; maybe it is possible to use right away?

BTW Thank you very much for working on this package, it's amazing!

**Comment by karthink** [comment]
No, it's not possible to use right away because this appears to not be related to one of the popular APIs gptel supports.

The documentation is also really sparse.  Any idea if they have a complete API reference?  I couldn't find one.

**Comment by karthink** [comment]
@zzantares If you are still interested in this feature please let me know if you find an API reference.

**Comment by zzantares** [comment]
Sure would be great! Here is the reference: https://developers.cloudflare.com/api/resources/ai/methods/run/
